packages/components/src/components/Badge/Badge.tsx

```diff
@@ -11,7 +11,7 @@ import { TransientProps } from '../../utils/transientProps';
 export const allowedBadgeFrameProps: FramePropsKeys[] = ['margin'];
 type AllowedFrameProps = Pick<FrameProps, (typeof allowedBadgeFrameProps)[number]>;
 
-type BadgeSize = Extract<UISize, 'tiny' | 'small' | 'medium'>;
+export type BadgeSize = Extract<UISize, 'tiny' | 'small' | 'medium'>;
```

Comment:
nit: 
You can import `BadgeProps` and use `BadgeProps['size']` and it does the same